docs/superpowers/plans/2026-04-29-stryker-core.md

```
@@ -0,0 +1,229 @@
+```bash
+dotnet tool install --global dotnet-stryker
+```
+
+Run the Core mutation test pass from the repository root:
+
+```bash
+dotnet stryker --config-file stryker-config.json
+```
+
+The initial Stryker setup is intentionally local-only. Use the HTML report to inspect surviving mutants and establish a baseline before adding CI thresholds or expanding the scope beyond Core.
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)